codalab/lib/spec_util.py

```diff
@@ -11,9 +11,7 @@
 UUID_PREFIX_REGEX = re.compile('^0x[0-9a-f]{1,31}$')
 BEGIN_NAME_STR = '[a-zA-Z_]'
 NAME_STR = '[a-zA-Z_][a-zA-Z0-9_\.\-]*'
-NAME_PATTERN_STR = (
-    '[%\*a-zA-Z0-9_\.\-]+'
-)  # Allow % for matching wildcard (SQL syntax), and * (regular expressions)
+NAME_PATTERN_STR = '[%\*a-zA-Z0-9_\.\-]+'  # Allow % for matching wildcard (SQL syntax), and * (regular expressions)
```

Comment:
This was accidentally formatted by a wrong version of black on my earlier hotfix commit.